scopes/preview/preview/preview.main.runtime.tsx

```diff
@@ -639,7 +641,9 @@ export class PreviewMain {
             return file.path;
           }
           const distRelativePath = compilerInstance.getDistPathBySrcPath(file.relative);
-          return join(this.workspace.path, modulePath, distRelativePath);
+          const moduleName = modulePath.replace(/^.*node_modules\//, '');
```

Comment:
I didn't get a module name but only `modulePath` to start with. So I convert it into module name in this way. I didn't find any way better to do this yet.

Reply:
What is the problem with the module path? it's relative not abs.
Using the module name like you did will not load it from the relevant .bit_root
which will lead to wrong resolution of peer dependencies.

Reply:
I see. it's because Vite can't recognize them as a file in an npm package so will fail to process them in cjs format. For dealing with potential issues about `.bit_root` I don't have any idea yet. I understand the file paths here are all overview and compositions entries for a component you would like to preview, not peer deps. Do you think it would cause any real issues?